rasa/shared/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -36,6 +37,8 @@
 KEY_LOOKUP = "lookup"
 KEY_LOOKUP_EXAMPLES = "examples"
 KEY_METADATA = "metadata"
+KEY_METADATA_INTENT = "intent"
+KEY_METADATA_EXAMPLE = "example"
```

Comment:
Can't we use the existing constants from [here](https://github.com/RasaHQ/rasa/blob/954e4734d479924df7211cd3bc5503285d54a66b/rasa/shared/nlu/constants.py#L11)?

Reply:
Ah, didn't know about this part. Will use those then ✅